### PR TITLE
PEP-8 cleanup (line spacing, white space, & imports)

### PR DESCRIPTION
**Pull request by leetrout**

### CODE REVIEW

File: zebra/forms.py [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
-    stripe_token  = forms.CharField(required=True, widget=forms.HiddenInput())
+    last_4_digits = forms.CharField(required=True, min_length=4, max_length=4, widget=forms.HiddenInput()  )
+    stripe_token = forms.CharField(required=True, widget=forms.HiddenInput())
 
 
 class StripePaymentForm(CardForm):
-    card_number         = forms.CharField(required=False, max_length=20, widget=NoNameTextInput())
-    card_cvv            = forms.CharField(required=False, max_length=4,  widget=NoNameTextInput())
-    card_expiry_month   = forms.ChoiceField(required=False, widget=NoNameSelect(), choices=MONTHS.iteritems())
-    card_expiry_year    = forms.ChoiceField(required=False, widget=NoNameSelect(), choices=options.ZEBRA_CARD_YEARS_CHOICES)
```
Comment (skoczen) [inline zebra/forms.py:23]: Everything else looks good, but this stuff I strongly disagree with.

I know it's PEP-8, but that tool isn't a substitute for our intelligence, and the neatly-formatted lines are much more readable.

Comment (leetrout) [inline zebra/forms.py:23]: That's not necessarily true. Beyond increasing line length it makes it more difficult for eyes to track across the lines quickly with the large amounts of white space between the identifier and the value.

With the added white space they do look more uniform which is aesthetically pleasing and easier to draw upon the similarities in the lines but I wouldn't say more readable.

Do as you wish after the merge. I'll say no more. If you want me to change it back before you merge I can do that, too.

Comment (skoczen) [inline zebra/forms.py:23]: Yeah, I get the read-across thing. I just think in most cases, you're scanning down for information (looking at field types, etc). 

I'll probably toss it back in post-merge, assuming I remember.  Formatting details aren't something I care that greatly about - I just wanted to let you know before randomly changing the formatting you worked on :)

Comment (leetrout) [inline zebra/forms.py:23]: Heh, if you went back and un-did something I'd get the message pretty quick ;)

Comment (skoczen) [inline zebra/forms.py:23]: Sure, but in a bit more of a rude way than I'd intend!

**Pull request merged**